runtime/scripts/parts/patternmatch.js

```diff
@@ -37,6 +37,8 @@ var PatternMatchPart = Numbas.parts.PatternMatchPart = function(xml, path, quest
 
 	settings.correctAnswerString = Numbas.xml.getTextContent(this.xml.selectSingleNode('correctanswer'));
 
+	//No idea what to do here.
+
```

Comment:
Load the `mode` attribute from the `correctanswer` tag into `this.settings.matchMode`:

    tryGetAttribute(settings,this.xml,'correctanswer',['mode'],['matchMode']);